openrndr-core/src/main/kotlin/org/openrndr/shape/Rectangle.kt

```diff
@@ -9,67 +9,74 @@ import kotlin.math.max
 import kotlin.math.min
 
 /**
- * Rectangle with double precision
+ * Creates a new [Rectangle].
+ *
+ * Also see [OrientedRectangle] and [IntRectangle].
  */
 data class Rectangle(val corner: Vector2, val width: Double, val height: Double = width) {
 
     constructor(x: Double, y: Double, width: Double, height: Double = width) :
             this(Vector2(x, y), width, height)
 
-    /** the center of the rectangle */
+    /** The center of the [Rectangle]. */
     val center: Vector2
         get() = corner + Vector2(width / 2, height / 2)
 
-    /** the unitless area covered by this rectangle*/
+    /** The unitless area covered by this [Rectangle]. */
```

Comment:
Why is it unitless?

Reply:
Because they don't have hardcoded units? They are not in square meters, square kilometers, etc... Whatever unit you assumed when creating the rectangle is what you get out. Should that be expressed in a different way?

Reply:
Any further arguments or can I set this to *resolved*?